README.rst

```diff
@@ -75,6 +69,40 @@ Every time you develop something in this repo
 
   # Open a PR and ask for review.
 
+Testing locally
+---------------
+
+* Install all dependencies using ``make requirements``
+* Run redis server via devstack by adding below section to ``docker-compose.yml`` file in devstack, this will bind redis to localhost port ``6379``.
+
+.. code-block:: yaml
+
```

Comment:
@bmtcril I'll create a `docker-compose.yml` file for having a test redis instance and update the docs.

Re the ResponseError, it only occurs when two consumers are started at the same time and the stream does not exists (both of them try to create it at the same time causing this issue). And, I was not able to reproduce the error consistently even with `make multiple_consumer_test_event` which creates them at almost same time. If you replace the command with `make consume_test_event` which creates a single consumer, it will work properly each time. I think the probability of users running two consumers at the same time for a topic which does not exist is very low, but I have wrapped it in try/catch and logged a warning.